runtime/calamari/src/lib.rs

```diff
@@ -204,7 +203,7 @@ impl pallet_authorship::Config for Runtime {
 }
 
 parameter_types! {
-	pub const NativeTokenExistentialDeposit: u128 = MA;
+	pub const NativeTokenExistentialDeposit: u128 = 10 * cMA; // 0.1 KMA
```

Comment:
Same comment as the previous.

Reply:
1MA to create a new account is really not reasonable.